<?xml version="1.0" ?>
<!--This does not replace URDF, and is not an extension of URDF.
    This is a format for representing semantic information about the robot structure.
    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
-->
<robot name="romeo">

  <group_state name="half_sitting" group="all">
        <joint name="root_joint" value="0. 0. 1. 0. 0. 0. 1." />
        <joint name="WAIST_P" value="1.0" />
        <joint name="WAIST_R"  value="1.0" />
        <joint name="CHEST"  value="0.0" />
        <joint name="LARM_SHOULDER_P"  value="0.0" />
        <joint name="LARM_SHOULDER_R"  value="0.0" />
        <joint name="LARM_SHOULDER_Y"  value="0.0" />
        <joint name="LARM_ELBOW"  value="0.0" />
        <joint name="LARM_WRIST_Y"  value="0.0" />
        <joint name="LARM_WRIST_P"  value="0.0" />
        <joint name="LARM_WRIST_R"  value="0.0" />
        <joint name="RARM_SHOULDER_P"  value="0.0" />
        <joint name="RARM_SHOULDER_R"  value="0.0" />
        <joint name="RARM_SHOULDER_Y"  value="0.0" />
        <joint name="RARM_ELBOW"  value="0.0" />
        <joint name="RARM_WRIST_Y"  value="0.0" />
        <joint name="RARM_WRIST_P"  value="0.0" />
        <joint name="RARM_WRIST_R"  value="0.0" />
        <joint name="LLEG_HIP_R"  value="0.0" />
        <joint name="LLEG_HIP_P"  value="0.0" />
        <joint name="LLEG_HIP_Y"  value="0.0" />
        <joint name="LLEG_KNEE"  value="0.0" />
        <joint name="LLEG_ANKLE_P"  value="0.0" />
        <joint name="LLEG_ANKLE_R"  value="0.0" />
        <joint name="RLEG_HIP_R"  value="0.0" />
        <joint name="RLEG_HIP_P"  value="0.0" />
        <joint name="RLEG_HIP_Y"  value="0.0" />
        <joint name="RLEG_KNEE"  value="0.0" />
        <joint name="RLEG_ANKLE_P"  value="0.0" />
        <joint name="RLEG_ANKLE_R"  value="0.0" />
    </group_state>

  <group_state name="half_sitting2" group="all">
        <joint name="root_joint" value="0. 0. 1. 0. 0. 0. 1." />
        <joint name="WAIST_P" value="-1.0" />
        <joint name="WAIST_R"  value="1.0" />
        <joint name="CHEST"  value="0.0" />
        <joint name="LARM_SHOULDER_P"  value="0.0" />
        <joint name="LARM_SHOULDER_R"  value="0.0" />
        <joint name="LARM_SHOULDER_Y"  value="0.0" />
        <joint name="LARM_ELBOW"  value="0.0" />
        <joint name="LARM_WRIST_Y"  value="0.0" />
        <joint name="LARM_WRIST_P"  value="0.0" />
        <joint name="LARM_WRIST_R"  value="0.0" />
        <joint name="RARM_SHOULDER_P"  value="0.0" />
        <joint name="RARM_SHOULDER_R"  value="0.0" />
        <joint name="RARM_SHOULDER_Y"  value="0.0" />
        <joint name="RARM_ELBOW"  value="0.0" />
        <joint name="RARM_WRIST_Y"  value="0.0" />
        <joint name="RARM_WRIST_P"  value="0.0" />
        <joint name="RARM_WRIST_R"  value="0.0" />
        <joint name="LLEG_HIP_R"  value="0.0" />
        <joint name="LLEG_HIP_P"  value="0.0" />
        <joint name="LLEG_HIP_Y"  value="0.0" />
        <joint name="LLEG_KNEE"  value="0.0" />
        <joint name="LLEG_ANKLE_P"  value="0.0" />
        <joint name="LLEG_ANKLE_R"  value="0.0" />
        <joint name="RLEG_HIP_R"  value="0.0" />
        <joint name="RLEG_HIP_P"  value="0.0" />
        <joint name="RLEG_HIP_Y"  value="0.0" />
        <joint name="RLEG_KNEE"  value="0.0" />
        <joint name="RLEG_ANKLE_P"  value="0.0" />
        <joint name="RLEG_ANKLE_R"  value="0.0" />
    </group_state>
    
    <rotor_params>
        <joint name="WAIST_P" mass="1e-5" gear_ratio="100." />
        <joint name="WAIST_R" mass="1e-5" gear_ratio="100.0" />
        <joint name="CHEST" mass="1e-5" gear_ratio="100." />
        <joint name="LARM_SHOULDER_P" mass="1e-5" gear_ratio="100." />
        <joint name="LARM_SHOULDER_R" mass="1e-5" gear_ratio="100." />
        <joint name="LARM_SHOULDER_Y" mass="1e-5" gear_ratio="100." />
        <joint name="LARM_ELBOW" mass="1e-5" gear_ratio="100." />
        <joint name="LARM_WRIST_Y" mass="1e-5" gear_ratio="100." />
        <joint name="LARM_WRIST_P" mass="1e-5" gear_ratio="100." />
        <joint name="LARM_WRIST_R" mass="1e-5" gear_ratio="100." />
        <joint name="RARM_SHOULDER_P" mass="1e-5" gear_ratio="100." />
        <joint name="RARM_SHOULDER_R" mass="1e-5" gear_ratio="100." />
        <joint name="RARM_SHOULDER_Y" mass="1e-5" gear_ratio="100." />
        <joint name="RARM_ELBOW" mass="1e-5" gear_ratio="100." />
        <joint name="RARM_WRIST_Y" mass="1e-5" gear_ratio="100." />
        <joint name="RARM_WRIST_P" mass="1e-5" gear_ratio="100." />
        <joint name="RARM_WRIST_R" mass="1e-5" gear_ratio="100." />
        <joint name="LLEG_HIP_R" mass="1e-5" gear_ratio="100." />
        <joint name="LLEG_HIP_P" mass="1e-5" gear_ratio="100." />
        <joint name="LLEG_HIP_Y" mass="1e-5" gear_ratio="100." />
        <joint name="LLEG_KNEE" mass="1e-5" gear_ratio="100." />
        <joint name="LLEG_ANKLE_P" mass="1e-5" gear_ratio="100." />
        <joint name="LLEG_ANKLE_R" mass="1e-5" gear_ratio="100." />
        <joint name="RLEG_HIP_R" mass="1e-5" gear_ratio="100." />
        <joint name="RLEG_HIP_P" mass="1e-5" gear_ratio="100." />
        <joint name="RLEG_HIP_Y" mass="1e-5" gear_ratio="100." />
        <joint name="RLEG_KNEE" mass="1e-5" gear_ratio="100." />
        <joint name="RLEG_ANKLE_P" mass="1e-5" gear_ratio="100." />
        <joint name="RLEG_ANKLE_R" mass="1e-5" gear_ratio="100." />
    </rotor_params>

</robot>
